appdev/guides/writeable-dirs.rst

```diff
@@ -0,0 +1,90 @@
+Writable directories
+====================
+
+App confinement is part of the Ubuntu Touch security concept. Data can be
```

Comment:
As mentioned in [the Documentation contribute guide](https://docs.ubports.com/en/latest/contribute/documentation.html#line-length), there is no need to break lines for arbitrary length.

Reply:
Done